config/routes.rb

```diff
@@ -1,12 +1,12 @@
 Rails.application.routes.draw do
-  get 'users/new'
   scope "(:locale)", locale: /en|vi/ do
   root "static_pages#home"
```

Comment:
sai indent

Reply:
ko sửa à em
fix tương tự bên dưới nhé
```suggestion
    root "static_pages#home"
```